src/main/java/org/sunbird/profile/service/ProfileService.java

```diff
@@ -18,7 +18,7 @@ public interface ProfileService {
 
 	SBApiResponse userBasicProfileUpdate(Map<String, Object> request);
 
-	SBApiResponse userAutoComplete(String searchTerm);
+	SBApiResponse userAutoComplete(String searchTerm, String rootOrgId);
```

Comment:
Let's have two methods... one with existing and new one

Reply:
2 seperate methods have been created.